CHANGELOG.md

```diff
@@ -8,6 +8,22 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 ## [Unreleased]
 
+The release adds logs support.
+Set `OTEL_LOGS_EXPORTER=otlp` to enable logs support.
+However, take notice that as for now
+the OpenTelemetry Go Logs API is not stable.
+
+### Added
+
+- Add logs support and `OTEL_LOGS_EXPORTER` environment variable.
+  `OTEL_LOGS_EXPORTER` accepts:
+  `none` - logs disabled,
+  `otlp` - OTLP gRPC exporter.
+  Currently `OTEL_LOGS_EXPORTER` defaults to `none`
+  as OpenTelemetry Go logs API and SDK are not stable yet.
+  Set `OTEL_LOGS_EXPORTER=otlp` to enable
+  logs support.
```

Comment:
Please reformat using the existing style of the document: wrap at an 80 character limit.